### PR TITLE
Removed button roles, fixed arialabel for select unselect all

### PR DESCRIPTION
**Pull request by BraydonKains**

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	"ariaSelectExemptUser": "Select {fullName} exempt",
 	"ariaSelectNotExemptUser": "Select {fullName} not exempt",
 	"ariaTableSummary": "The list of users that can have their exemption status modified.",
+	"ariaTableCaption": "I would really like this to read to the screen",
```
Comment (BraydonKains) [inline lang/en.json]: Fixing this right now

Comment (BraydonKains) [reply lang/en.json]: Does this even need to say anything different than the summary? Maybe we should use the summary in both places or just remove the d2l-offscreen. I never got the screenreader to pick this up anyway.

File: d2l-activity-exemptions.js [hunk 2]
```diff
@@ -105,6 +103,7 @@ class D2LActivityExemptions extends mixinBehaviors(
 		  id="classlist"
 		  role="grid"
 		  summary="[[localize('ariaTableSummary')]]"
+		  aria-label$="[[localize('ariaTableSummary')]]"
 		  sticky-headers=""
 		>
 		  <d2l-offscreen>[[localize('ariaTableCaption')]]</d2l-offscreen>
```
Comment (jwong-d2l) [inline d2l-activity-exemptions.js]: Is this a lang term?  Is this different from the aria-label for the grid?

File: lang/en.json (hunk 1 above)
Comment (jwong-d2l) [reply lang/en.json]: I think removing the offscreen is the way to go.

**Pull request merged**